### ISSUE TITLE
accessibility iteration

### ISSUE THREAD

**Issue by devCrossNet** [devCrossNet/vue-starter#259]
not all components are optimized for accessibility at the moment, we should take another round on all of them and see how we can improve them.

**Comment by devCrossNet** [comment]
Tab should be accessible via keyboard
